tests/test_validate_config.py

```diff
@@ -57,6 +57,16 @@ def test_no_dashboard_password(self):
         PillarManager.reset('ceph-salt:dashboard:password')
         self.assertEqual(validate_config([]), "No dashboard password specified in config")
 
+    def test_dashboard_ssl_certificate(self):
+        PillarManager.reset('ceph-salt:dashboard:ssl_certificate_key')
+        self.assertEqual(validate_config([]), "Dashboard SSH certificate provided, "
+                                              "but no SSH certificate key specified")
+
+    def test_dashboard_ssl_certificate_key(self):
+        PillarManager.reset('ceph-salt:dashboard:ssl_certificate')
+        self.assertEqual(validate_config([]), "Dashboard SSH certificate key provided, "
+                                              "but no SSH certificate specified")
```

Comment:
this code change mentions "SSH certificate" four times - should be "SSL certificate" in all cases I think

Reply:
Fixed